micro-rdk-server/native/main.rs

```diff
@@ -41,10 +43,14 @@ mod native {
         // At runtime, if the program does not detect credentials or configs in storage,
         // it will try to load statically compiled values.
 
-        if !storage.has_robot_configuration() {
+        if !storage.has_robot_credentials() {
```

Comment:
I believe that this was a bug. The other two mainlines here check `credentials`, but `native` was checking `configuration`.